pkg/collector/corechecks/cluster/ksm/kubernetes_state_defaults.go

```diff
@@ -227,3 +229,10 @@ func getLabelToMatchForKind(kind string) []string {
 		return []string{kind, "namespace"}
 	}
 }
+
+func defaultAnnotationsAsTags() map[string]map[string]string {
```

Comment:
how does it work for the unified service tagging? it is because in the case of UST we used labels?

Reply:
UST are labels, and defined in `defaultLabelJoins` . It doesn't interfere at all.